solutions/cf-1359C.mdx

```diff
@@ -0,0 +1,114 @@
+---
+id: cf-1359C
+source: CF
+title: Mixing Water
+author: Jesse Choe
+---
+
+<Info title="Constant Time Solution">
+**Note: the $\mathcal{O}(1)$ solution for this problem is provided in the
+[official editorial](https://codeforces.com/blog/entry/78116).**
+</Info>
+
+Consider the following observations:
+
+- Case 1: pouring an equal number of cups of hot and cold water is equivalent to pouring one cup of each. Otherwise,
+- Case 2: there must be exactly one more cup of hot water poured than cold water poured. Thus, there will be $c_i + 1$ cups of hot water and $c_i$ cups of cold water for some $c \geq 0$.
```

Comment:
these are not really observations or cases (i think the first is an observation and the second is a case)...maybe
```suggestion
- Case 1: The number of cups of hot water poured is equal to the number of cups of cold water poured. Pouring an equal number of cups of hot and cold water is equivalent to pouring one cup of each. 
- Case 2: Otherwise, there must be exactly one more cup of hot water poured than cold water poured. Thus, there will be $c_i + 1$ cups of hot water and $c_i$ cups of cold water for some $c_i \geq 0$.
```